pkg/plugins/backendplugin/grpcplugin/client_v2.go

```diff
@@ -52,6 +52,11 @@ func newClientV2(descriptor PluginDescriptor, logger log.Logger, rpcClient plugi
 		return nil, err
 	}
 
+	rawVCS, err := rpcClient.Dispense("versionedstorage")
```

Comment:
```suggestion
	rawVCS, err := rpcClient.Dispense("vcs")
```

I got an error during plugins initialization because it seems this constant needs to match the plugin type (and it has been defined as "vcs" on the plugin types map (see the file above).

Reply:
I wasn't sure about that because I thought we could satisfy several client interfaces with one plugin. Like "resource" is defined here: https://github.com/grafana/grafana/blob/main/pkg/plugins/backendplugin/grpcplugin/client_v2.go#L35 and seems defined here for the renderer plugin: https://github.com/grafana/grafana-image-renderer/blob/master/proto/pluginv2.proto#L63-L89, so I thought it relied on the service name. But I must admit that I don't understand 100% of what's going on there.